core/services/fluxmonitor/flux_monitor.go

```diff
@@ -689,29 +592,29 @@ func (p *PollingDeviationChecker) createJobRun(nextPrice decimal.Decimal, nextRo
 var dec0 = decimal.NewFromInt(0)
 
 // OutsideDeviation checks whether the next price is outside the threshold.
-func OutsideDeviation(curPrice, nextPrice decimal.Decimal, threshold float64) bool {
-	if curPrice.Equal(dec0) {
-		logger.Infow("Current price is 0, deviation automatically met", "currentPrice", dec0)
+func OutsideDeviation(curAnswer, nextAnswer decimal.Decimal, threshold float64) bool {
+	if curAnswer.Equal(dec0) {
+		logger.Infow("Current price is 0, deviation automatically met", "answer", dec0)
```

Comment:
should this maybe rethought and instead automatically poll only if the currentRound is 0? I think the purpose here is to automatically poll on the first round, when the current answer is 0... but in the future I imagine we may want to consider 0 values as potentially valid instead of trying to re-poll.

Reply:
Maybe so, but I left this logic identical to how I found it.  Maybe throw a story in tracker?

Reply:
Done.
https://www.pivotaltracker.com/story/show/171869391